### DISCUSSION TITLE
Is it possible to disable Atropisomer chirality?

### DISCUSSION THREAD

**Discussion by nadiahfl** [rdkit/rdkit#7897] [Q&A]
Hi! I'm trying to check the stereochemistry information on a chemical library and found the atropisomer chirality for macrocycles were reported. The example below returns [(10, '?')]). I don't need atropisomer chirality information, and the ideal return is [] for achiral. If I change includeUnassigned=False, the example below will return [], but I need to keep this argument to be True for other molecule which has true unassigned stereocenters. I wonder if there is an option to disable the atropisomer chirality detection. 

mol = Chem.MolFromSmiles('Clc1ccc2nc1Nc1cn(CCCOC2)cc1')
Chem.FindPotentialStereo (mol, cleanIt=True)
Chem.FindMolChiralCenters(mol,includeUnassigned=True)

**Comment by greglandrum** [comment]
@nadiahfl I cannot reproduce this behavior with either the 2024.09.1 or 2024.03.6 versions of the rdkit. Which version are you using?

As an FYI: `FindMolChiralCenters()` returns atomic chiral centers, not atropisomeric bonds, so even if there were atropisomeric bonds being detected here (there are not), they would not be returned by that function.

**Reply by nadiahfl** [reply]
Thanks, @greglandrum. I'm using 2023.09.4. It looks that it won't be an issue in the newer version, and will upgrade to newer version to test out. Yes, returning atomic chiral centers would be more expected. Thanks a lot for your help.
